src/js/components/TabButtons.js

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames/dedupe';
+import React from 'react';
+
+class TabButtons extends React.Component {
```

Comment:
Let's figure out a way to differentiate component names so it is not only differentiated by `s`